main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"challenge/middleware"
+	"challenge/webserver"
+	"log"
+	"net/http"
+)
+
+func main() {
```

Comment:
Do you know/have you used tools that help you with structuring CLI apps that help you with ENV parsing, config read/parse and so on?

Reply:
I am aware of Cobra for structuring/handling CLI apps. I've mainly used Viper for env/config reading/parsing for multiple components i had to develop.